lib/webauto/web4cucumber.rb

```diff
@@ -132,6 +132,9 @@ def browser
         end
       elsif @browser_type == :chrome
         logger.info "Launching Chrome"
+
+	#https://bugs.chromium.org/p/chromium/issues/detail?id=1056073
+	chrome_caps[:acceptInsecureCerts] = true
```

Comment:
Bad indention ?

Reply:
Strange, well indention in my local editor. Updated the bad indention on web directly.